serverless.yml

```
@@ -36,5 +36,5 @@ custom:
         - SALT
         - NODEMAILER_USER
         - NODEMAILER_PASS
-        - AWS_BUCKET_NAME
+        #  - AWS_BUCKET_NAME
```

```suggestion
        - AWS_BUCKET_NAME
```
추후 배포시에는 필요한 부분이라 개발시에 주석처리 한 부분은 빼주세요.

---

미처 확인하지못한점이네요.
감사합니다 :D